### ISSUE TITLE
redditinsight.com is down

### ISSUE THREAD

**Issue by AlleLouis** [PatrickJS/Reddit-Insight#31]
The redditinsight.com site has been down for the past few weeks showing the following error:

```
An error occurred in the application and your page could not be served. If you are the application owner, check your logs for details. You can do this from the Heroku CLI with the command
```

Do you plan on restoring the site or are you waiting for someone else to take it from here?

**Comment by Codarus** [comment]
Any update on this? Does anyone know if it's a simple hosting issue or something more?

**Comment by AlleLouis** [comment]
I contacted the maintainer 2 years ago and pointed out that the site was down. The maintainer confirmed reception of the message, but there was no follow up to hint the plans about the site. My understanding was that redditinsight.com was dropped or the fix was postponed indefinitely.

**Comment by googamanga** [comment]
The site is hosted on Heroku which has made breaking changes in its platform. The site has not been maintained for many years.

**Comment by ooker777** [comment]
can we host it in our own?
@PatrickJS can you have a look?

**Comment by PatrickJS** [comment]
yeah this whole app needs to be remade. I think the tracking is the main feature that people keep using

**Comment by ooker777** [comment]
So is there an old version that still works fine? I clone the [version 1.0.0](https://github.com/PatrickJS/Reddit-Insight/tree/v1.0.0) and it gives `TypeError: Cannot read property 'databaseName' of undefined`
